datafusion/functions/src/string/btrim.rs

```diff
@@ -16,27 +16,27 @@
 // under the License.
 
 use arrow::array::{ArrayRef, OffsetSizeTrait};
-use arrow::datatypes::DataType;
-use datafusion_common::exec_err;
-use datafusion_common::Result;
-use datafusion_expr::ColumnarValue;
-use datafusion_expr::TypeSignature::*;
-use datafusion_expr::{ScalarUDFImpl, Signature, Volatility};
 use std::any::Any;
 
-use crate::string::{make_scalar_function, utf8_to_str_type};
+use arrow::datatypes::DataType;
 
-use super::{general_trim, TrimType};
+use datafusion_common::{exec_err, Result};
+use datafusion_expr::TypeSignature::*;
+use datafusion_expr::{ColumnarValue, Volatility};
+use datafusion_expr::{ScalarUDFImpl, Signature};
 
-/// Returns the longest string  with leading and trailing characters removed. If the characters are not specified, whitespace is removed.
+use crate::string::common::*;
+
+/// Returns the longest string with leading and trailing characters removed. If the characters are not specified, whitespace is removed.
 /// btrim('xyxtrimyyx', 'xyz') = 'trim'
-pub fn btrim<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
+fn btrim<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
     general_trim::<T>(args, TrimType::Both)
 }
 
 #[derive(Debug)]
 pub(super) struct TrimFunc {
```

Comment:
It isn't a huge deal, but I found it a little confusing that the function is called `TrimFunc` but has a name of `"btrim"` and an alias of `"trim"`

I would expect either `BTrimFunc` or else the name of "trim" and an alias of `"btrim"`

Reply:
Urgh, this was an oversight on my part when moving the Trim function here. I'll fix.